src/reducers/searchResultsReducer.js

```diff
@@ -69,36 +69,39 @@ export default function course(state = initialState.SearchResults, {payload, typ
     }
 }
 
-const handleAccountSearchResults = (state, payload, status) =>{
-    let {response}= payload;
-    let {data} = response;
-
+const handleAccountSearchResults = (state, payload, status) => {
+    let { response } = payload;
+    let { data } = response;
+    console.log(data)
+    // you can get page and count
     return {
         ...state,
-        accounts:data.results,
+        count: data.count,
```

Comment:
there eventually will be 2 counts. this count is specifically for the number of account search results. When @tigermli updates the course search endpoint, you'll also need to track course count.